wporg-gp-translation-events.php

```diff
@@ -213,18 +213,40 @@ function submit_event_ajax() {
 		);
 		$response_message = 'Event updated successfully!';
 	}
+	if ( 'delete_event' === $_POST['form_name'] ) {
```

Comment:
I think we should use `$action` instead of `$_POST['form_name']`. `$action` has been sanitized.

Reply:
Done [here](https://github.com/WordPress/wporg-gp-translation-events/pull/69/commits/9fd3d6282b5160da2aa4d38d8ea34a31a127a377). Thank you.